2021-02-16 01:03:48,596 ERROR: Exception on /explore [GET] [in /Users/cooper/Desktop/Websites/microblog/venv/lib/python3.9/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/Users/cooper/Desktop/Websites/microblog/venv/lib/python3.9/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/cooper/Desktop/Websites/microblog/venv/lib/python3.9/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/cooper/Desktop/Websites/microblog/venv/lib/python3.9/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/cooper/Desktop/Websites/microblog/venv/lib/python3.9/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/cooper/Desktop/Websites/microblog/venv/lib/python3.9/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/cooper/Desktop/Websites/microblog/venv/lib/python3.9/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/cooper/Desktop/Websites/microblog/venv/lib/python3.9/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "/Users/cooper/Desktop/Websites/microblog/app/routes.py", line 47, in explore
    posts = Post.query.order_by(Post.timestamp.desc()).paginate(
  File "/Users/cooper/Desktop/Websites/microblog/venv/lib/python3.9/site-packages/flask_sqlalchemy/__init__.py", line 484, in paginate
    if page < 1:
TypeError: '<' not supported between instances of 'type' and 'int'
2021-02-16 01:03:51,556 ERROR: Exception on /explore [GET] [in /Users/cooper/Desktop/Websites/microblog/venv/lib/python3.9/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/Users/cooper/Desktop/Websites/microblog/venv/lib/python3.9/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/cooper/Desktop/Websites/microblog/venv/lib/python3.9/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/cooper/Desktop/Websites/microblog/venv/lib/python3.9/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/cooper/Desktop/Websites/microblog/venv/lib/python3.9/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/cooper/Desktop/Websites/microblog/venv/lib/python3.9/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/cooper/Desktop/Websites/microblog/venv/lib/python3.9/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/cooper/Desktop/Websites/microblog/venv/lib/python3.9/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "/Users/cooper/Desktop/Websites/microblog/app/routes.py", line 47, in explore
    posts = Post.query.order_by(Post.timestamp.desc()).paginate(
  File "/Users/cooper/Desktop/Websites/microblog/venv/lib/python3.9/site-packages/flask_sqlalchemy/__init__.py", line 484, in paginate
    if page < 1:
TypeError: '<' not supported between instances of 'type' and 'int'
2021-02-16 01:04:07,175 INFO: Microblog startup [in /Users/cooper/Desktop/Websites/microblog/app/__init__.py:51]
2021-02-16 01:35:42,599 INFO: Microblog startup [in /Users/cooper/Desktop/Websites/microblog/app/__init__.py:55]
2021-02-16 01:43:08,358 INFO: Microblog startup [in /Users/cooper/Desktop/Websites/microblog/app/__init__.py:55]
2021-02-16 01:45:58,753 INFO: Microblog startup [in /Users/cooper/Desktop/Websites/microblog/app/__init__.py:55]
